SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+All versions including and above the current stable release version number (the version downloadable on https://brave.com/download).
```

Comment:
Might be better worded as `Channels` instead of versions. When worded as `Versions`, it sounds like there would be an archive of old Brave releases

Reply:
People have sometimes opened HackerOne reports with old Brave versions (because they forgot to update) so I think versions helps clarify that old stable/beta/etc. are out of scope if they are older than current stable version number.